host/cli/bootstrap.go

```diff
@@ -963,16 +922,52 @@ DELETE FROM volumes WHERE created_at < '%s';`,
 			}
 		}
 		steps := bootstrap.Manifest{
-			manifestStepMap["docker-receive-secret"],
-			manifestStepMap["docker-receive"],
-			manifestStepMap["docker-receive-route"],
-			manifestStepMap["docker-receive-wait"],
+			manifestStepMap["tarreceive"],
+			manifestStepMap["tarreceive-route"],
+			manifestStepMap["tarreceive-wait"],
 		}
 		if _, err := steps.RunWithState(ch, state); err != nil {
-			return fmt.Errorf("error deploying docker-receive: %s", err)
+			return fmt.Errorf("error deploying tarreceive: %s", err)
```

Comment:
Should we delete the docker-receive app?

Reply:
It gets deleted below (see [here](https://github.com/flynn/flynn/blob/tarreceive/host/cli/bootstrap.go#L934-L969)).